example/lib/generated/messages_all_locales.dart

```diff
@@ -2,12 +2,8 @@
 // This is a library that looks up messages for specific locales by
 // delegating to the appropriate library.
 
-// Ignore issues from commonly used lints in this file.
-// ignore_for_file:implementation_imports, file_names
-// ignore_for_file:unnecessary_brace_in_string_interps, directives_ordering
-// ignore_for_file:argument_type_not_assignable, invalid_assignment
-// ignore_for_file:prefer_single_quotes, prefer_generic_function_type_aliases
-// ignore_for_file:comment_references
+// Ignore issues from any lints in this file.
+// ignore_for_file: type=lint, unused_import
```

Comment:
Generally I try for generated code to ~match hand-written code, both in readability and in the lints it passes (though I don't know how achievable this is in practice). I think a blanket `type=lint` would make sense for these files. If we decide to re-invest in this style of workflow we can re-visit the blanket ignore.